content/snippets/easy_to_change.md

```diff
@@ -0,0 +1,32 @@
+title: Easy to Test, Easy To Change
+tags: software engineering
+author: cdent
+created: 2024-02-23
+
+In conversation with my wife recently I mentioned that I've been doing software
+development for more than 40 years. I've learned, forgotten, and re-learned
+many things in that time.
+
+There are two goals I now accept as true and self evident but are neither
```

Comment:
```suggestion
There are two goals I now accept as true and self evident but which are neither
```

Reply:
this change is perhaps grammatically correct, but I reject it in terms of tone. "which" is something to be avoided where possible.

Reply:
heh alright